### PR TITLE
Update Node.js in workflows and fix release-please config

### PR DESCRIPTION
**Pull request by targos**
- **chore: use Node.js 20 for node-gyp integration tests**
- **chore: fix release-please config**

### CODE REVIEW

**Comment by targos** [comment]
Rebased to use the pinned Ruff version.

File: .github/workflows/release-please.yml [hunk 1]
```diff
@@ -11,6 +11,4 @@ jobs:
       - uses: google-github-actions/release-please-action@v4
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          release-type: python
-          package-name: gyp-next
-          bump-minor-pre-major: true
+          config-file: release-please-config.json
```
Comment (cclauss) [inline .github/workflows/release-please.yml:14]: Why is it useful to put the release context in two separate files?

Comment (targos) [reply .github/workflows/release-please.yml:14]: It's not useful, we have no choice as it's mandatory since the version 4 of the action.

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
Oh… https://github.com/nodejs/gyp-next/actions/workflows/release-please.yml

**Comment by targos** [comment]
Mmh, it wasn't obvious from the docs that both file must be there. I'll have a closer look.

**Comment by lukekarrys** [comment]
We just did the same upgrade in `node-gyp`. I believe the only thing you will need is the top level `.release-please-manifest.json` file with the following contents:

```json
{
    ".": "0.16.2"
}
```

File: .github/workflows/release-please.yml (hunk 1 above)
Comment (lukekarrys) [reply .github/workflows/release-please.yml:14]: You could also remove this option since that is also the default.

**Pull request merged**